### ISSUE TITLE
Failed to start Secret Service: Portal(CancelledPortalRequest)

### ISSUE THREAD

**Issue by wcasanova** [marhkb/pods#678]
OS:  Spin Fedora 38 KDE
pods:  v1.2.0 (46871b80)

```
flatpak run --env=RUST_BACKTRACE=full com.github.marhkb.Pods
thread 'main' panicked at 'Failed to start Secret Service: Portal(CancelledPortalRequest)', src/main.rs:95:14
stack backtrace:
   0:     0x55fcc42ccf80 - <unknown>
   1:     0x55fcc41d521e - <unknown>
   2:     0x55fcc42a80b4 - <unknown>
   3:     0x55fcc42ce62f - <unknown>
   4:     0x55fcc42ce22f - <unknown>
   5:     0x55fcc42cf2c8 - <unknown>
   6:     0x55fcc42ced84 - <unknown>
   7:     0x55fcc42cece6 - <unknown>
   8:     0x55fcc42cecd1 - <unknown>
   9:     0x55fcc3b2e1c2 - <unknown>
  10:     0x55fcc3b2e552 - <unknown>
  11:     0x55fcc417a0ba - <unknown>
  12:     0x55fcc4176009 - <unknown>
  13:     0x55fcc3c09593 - <unknown>
  14:     0x55fcc41c4309 - <unknown>
  15:     0x7f53cb22954a - <unknown>
  16:     0x7f53cb22960b - __libc_start_main
  17:     0x55fcc3b3f6f5 - <unknown>
  18:                0x0 - <unknown>
  ```

**Comment by marhkb** [comment]
I've released 1.2.1. Maybe you can give it a try.

**Comment by wcasanova** [comment]
yes, it works.
